lightning/restore/checkpoints.go

```diff
@@ -536,9 +542,10 @@ func (cpdb *MySQLCheckpointsDB) InsertEngineCheckpoints(ctx context.Context, tab
 				return errors.Trace(err)
 			}
 			for _, value := range engine.Chunks {
+				colPerm, _ := json.Marshal(value.ColumnPermutation)
 				_, err = chunkStmt.ExecContext(
 					c, tableName, engineID,
-					value.Key.Path, value.Key.Offset, value.Columns, value.ShouldIncludeRowID,
+					value.Key.Path, value.Key.Offset, colPerm,
```

Comment:
I only know we call `InsertEngineCheckpoints` after calling `populateChunks`.  If so, `columns` column always empty

Reply:
Removed this and the checksum (always 0 too).